src/Symfony/Bridge/Monolog/Command/ServerLogCommand.php

```diff
@@ -60,7 +61,7 @@ protected function configure()
             ->addOption('format', null, InputOption::VALUE_REQUIRED, 'The line format', ConsoleFormatter::SIMPLE_FORMAT)
             ->addOption('date-format', null, InputOption::VALUE_REQUIRED, 'The date format', ConsoleFormatter::SIMPLE_DATE)
             ->addOption('filter', null, InputOption::VALUE_REQUIRED, 'An expression to filter log. Example: "level > 200 or channel in [\'app\', \'doctrine\']"')
-            ->setDescription('Starts a log server that displays logs in real time')
+            ->setDescription(self::$defaultDescription)
```

Comment:
IMHO this could be done in the `Command::__construct` method:
- reduce bloating code
- prevent user's mistake that don't use the FrameworkBundle, and don't call `setDescription`

Reply:
I don't want to bump the console requirement of all components, nor to change the signature of the constructor.

Reply:
I still did what you suggested (calling setDescription()) in the constructor,
but I'm keeping all these calls to preserve compact with older versions of symfony/console